android/java/res/layout/brave_permission_prompt_dialog.xml

```diff
@@ -11,13 +11,27 @@
     android:gravity="start"
     style="@style/AlertDialogContent">
 
-    <ImageView
-        android:id="@+id/favicon"
-        android:layout_height="60dp"
-        android:layout_width="60dp"
-        android:layout_gravity="center_horizontal"
-        android:contentDescription="@null"
-        android:visibility="gone"/>
+    <com.google.android.material.card.MaterialCardView
+            android:id="@+id/image_container_card"
+            android:layout_width="60dp"
+            android:layout_height="60dp"
+            android:layout_margin="10dp"
+            android:elevation="2dp"
```

Comment:
nit: same, elevation and border may not required per design and desktop, cross check and also try changing to dark theme.